app/build.gradle

```diff
@@ -56,7 +56,7 @@ dependencies {
     implementation 'androidx.activity:activity-compose:1.7.2'
     implementation "androidx.compose.ui:ui:$compose_version"
     implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
-    implementation 'androidx.compose.material3:material3:1.2.0-alpha02'
+    implementation 'androidx.compose.material3:material3:1.1.1'
```

Comment:
we downgrading this package?

Reply:
yes to stable lol

Reply:
i also have no clue what navigation runtime is for? in the dependencies